helm-charts/falcon-kac/templates/deployment_webhook.yaml

```diff
@@ -144,7 +145,11 @@ spec:
         - containerPort: {{ .Values.webhookPort }}
           name: webhook-port
         resources:
+          {{- if .Values.admissionControl.enabled }}
```

Comment:
Don't you also need a similar `{{- if .Values. visibilityEnabled.enabled }}` for falcon-watcher container

Reply:
No, this is just for the falcon-client container. The idea is to allow to disable just the admission control functionality.

Reply:
I'd gladly add the change to the falcon-watcher container resources when visibility is disabled. Though, come to think of it, it's outside the scope of this PR. I'd rather make a separate PR with that change.